src/style.css

```diff
@@ -1 +1,120 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir.ttc');
+  font-weight: normal;
```

Comment:
Remove redundant default value (400 it's default too)
```suggestion
```

But for this task should be another weight so please fix it 
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/70009205/cfaa0e04-e388-4f15-8146-1064b59eaf3e)